meiki_server/notes/routes.go

```diff
@@ -7,18 +7,45 @@ import (
 	"github.com/gin-gonic/gin"
 )
 
-const MSG_UNABLE_TO_CREATE_NOTE = "Unable to create note"
+const MSG_UNABLE_TO_CREATE_NOTE = "Unable to create note, please try again later"
 const MSG_PARSE_ERROR = "Unable to parse request body"
+const MSG_INVALID_TITLE = "Invalid note title"
+const MSG_NOTE_DOES_NOT_EXIST = "Note does not exist"
+const MSG_INVALID_ID = "Invalid id"
+const MSG_UNABLE_TO_PERFORM_ACTION = "Unable perform requested action"
 
 type CreateRequest struct {
-	Title    string `json:"title"`
-	UserName string `json:"username"`
+	Title string `json:"title"`
 }
 
+type RenameRequest = CreateRequest
```

Comment:
I think the abstraction is misleading here. At best both of this should a NewRequest type

Reply:
`TitleRequest`?

Reply:
yeah that works